tests/unit/fs/test_http.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+from dvc_http import HTTPFileSystem
+
+url = "https://remote.dvc.org/get-started"
+
+
+@pytest.mark.parametrize(
+    "kwarg,value",
+    (
+        ("connect_timeout", 42),
+        ("read_timeout", 42),
+    ),
+)
+def test_timeout_options(dvc, kwarg, value):
+    config = {"url": url, kwarg: value}
+    fs = HTTPFileSystem(**config)
+
+    assert fs.fs_args["client_kwargs"][kwarg] == value
```

Comment:
This test probably belongs in `dvc-http` as it is not testing anything specific about dvc (even the config is directly passed to `HTTPFileSystem`).

Reply:
Yeah, to be moved `dvc-http`. Also see #8769